src/components/QueryEditor.tsx

```diff
@@ -221,6 +229,11 @@ export class QueryEditor extends PureComponent<Props, State> {
             />
           </div>
         )}
+        <div className={'gf-form-inline'}>
+          <QueryField label="Render after all queries finish" labelWidth={12}>
```

Comment:
Is this a bit misleading? Even though multiple pages are being returned, it's still only one query?

Reply:
so this PR is doing two things:
 - Return several pages for a single query (all backend stuff)
 - If there are several queries, and one of them has this toggle enabled, it will wait for all queries before rendering the result (which is the default). See [this line](https://github.com/grafana/timestream-datasource/pull/133/files#diff-744ba81e1aaee1ed9a56abf8a987f391f665e20d9ed2298b757b2edf6dc8752fR102). That's the reason for this label.

Reply:
Yeah I saw that - see what you mean now. I wish it was possible for the plugin to specify request options, i.e on a batch level.